moveit_ros/hybrid_planning/test/hybrid_planning_demo_node.cpp

```diff
@@ -55,7 +55,11 @@
 #include <rclcpp/node_options.hpp>
 #include <rclcpp/parameter_value.hpp>
 #include <rclcpp/qos.hpp>
+#if MOVEIT_ROLLING
 #include <rclcpp/event_handler.hpp>
+#elif MOVEIT_HUMBLE
+#include <rclcpp/qos_event.hpp>
+#endif
```

Comment:
```suggestion
#if MOVEIT_ROLLING
  #include <rclcpp/event_handler.hpp>
#elif MOVEIT_HUMBLE
  #include <rclcpp/qos_event.hpp>
#endif
```

Reply:
It's automatically formatted with clang-format, CI will fail if I apply this patch